tasks/release.py

```diff
@@ -117,3 +122,255 @@ def _find_v6_tag(ctx, v7_tag):
+            if _is_version_higher(this_version, highest_version):
+                highest_version = this_version
+    return highest_version
+
+
+def _get_highest_version_from_release_json(release_json, highest_major, version_re):
+    highest_version = None
+    highest_jmxfetch_version = None
+    for key, value in release_json.items():
+        match = version_re.match(key)
+        if match:
+            this_version = _create_version_dict_from_match(match)
+            if _is_version_higher(this_version, highest_version) and this_version["major"] <= highest_major:
+                match = version_re.match(value["JMXFETCH_VERSION"])
+                if match:
+                    highest_jmxfetch_version = _create_version_dict_from_match(match)
+                    highest_version = this_version
+                else:
+                    print("{} does not have a valid JMXFETCH_VERSION ({}), ignoring".format(_stringify_version(this_version), value["JMXFETCH_VERSION"]))
+    return highest_version, highest_jmxfetch_version
+
+
+def _save_release_json(release_json, list_major_versions, highest_version, integration_version, omnibus_software_version, omnibus_ruby_version, jmxfetch_version):
+
+    jmxfetch = urllib.urlopen("https://bintray.com/datadog/datadog-maven/download_file?file_path=com%2Fdatadoghq%2Fjmxfetch%2F{}%2Fjmxfetch-{}-jar-with-dependencies.jar")
+    jmxfetch_sha256 = hashlib.sha256(jmxfetch.read()).hexdigest()
+
+    print("Jmxfetch's SHA256 is {}".format(jmxfetch_sha256))
+
+    new_version_config = OrderedDict()
+    new_version_config["INTEGRATIONS_CORE_VERSION"] = integration_version
+    new_version_config["OMNIBUS_SOFTWARE_VERSION"] = omnibus_software_version
+    new_version_config["OMNIBUS_RUBY_VERSION"] = omnibus_ruby_version
+    new_version_config["JMXFETCH_VERSION"] = jmxfetch_version
+    new_version_config["JMXFETCH_HASH"] = jmxfetch_sha256
+
+    # Necessary if we want to maintain the JSON order, so that humans don't get confused
+    new_release_json = OrderedDict()
+
+    # The nightlies should be at the top of the file
+    nightly_version_re = re.compile('nightly(-a[0-9])?')
+    for key, value in release_json.items():
+        if nightly_version_re.match(key):
+            new_release_json[key] = value
+
+    # Then the new versions
+    for version in list_major_versions:
+        highest_version["major"] = version
+        new_version = _stringify_version(highest_version)
+        new_release_json[new_version] = new_version_config
+
+    # Then the rest of the versions
+    for key, value in release_json.items():
+        if key not in new_release_json:
+            new_release_json[key] = value
+
+    with open("release.json", "w") as release_json_stream:
+        json.dump(new_release_json, release_json_stream, indent=4, sort_keys=False)
+
+
+@task
+def create_new_version(ctx, major_versions = "6,7", omnibus_ruby_version = "datadog-5.5.0"):
```

Comment:
should probably have command line args for `omnibus_software_version`, which we should also tag.
And probably `integrations-core`, too.